service/mantle/shopify/ShopifyServices.xml

```diff
@@ -131,4 +131,44 @@
             <set field="locationId" from="productLocation.settingValue" />
         </actions>
     </service>
+    <service verb="validate" noun="ProductData">
+        <description>
+            This service validates if the data of the product is correcto for using in a shopifyRequest

```

Comment:
Change the word *correcto*
            This service validates if the data of the product is valid to use in a shopify request

Reply:
The correction was applied